app/controllers/finders_controller.rb

```diff
@@ -84,6 +85,14 @@ def result_set_presenter
     )
   end
 
+  def results
+    @results ||= ResultSetParser.parse(
+      search_results.fetch("results"),
+      search_results.fetch("start", 0),
```

Comment:
Is this right?  In `FinderPresenter` the value was `search_results.fetch('start', 0) + 1`